### DISCUSSION TITLE
Integrating multiple samples from 10X multiome data? Is there a tutorial for that?

### DISCUSSION THREAD

**Discussion by micosacak** [stuart-lab/signac#1587] [Q&A]
Hi,

It is not clear for me how to integrate multiple samples of 10X multiome data using SigNac? Is there a tutorial for that? For instance, here the integration is explained (https://stuartlab.org/signac/articles/pbmc_multiomic) or here (https://satijalab.org/seurat/articles/seurat5_atacseq_integration_vignette), but it is for one sample. What if there are conditions (control vs treatment)?

to make my question simple; let's assume we have 2 samples (control vs treatment). And we have 10X multiome data for both samples.  How can I analyses these data together? Is there an example tutorial that I missed or may be a simple trick of code for that?

thanks,
ilyas.

**Comment by timoast** [comment]
Hi @micosacak, we don't have a vignette for this specifically but you can take a look at some of the previous discussion here:
https://github.com/stuart-lab/signac/discussions/438
https://github.com/stuart-lab/signac/issues/824

At some point we will try to put together a vignette for this since it's becoming a common question
